### PR TITLE
Update info files

### PR DESCRIPTION
**Pull request by rs028**
Address issue #10

### CODE REVIEW

**Comment by rs028** [comment]
I am not sure we need to keep `diff_version_rolf_sander.txt`. It seems to be just the output of a diff command with no explanation about the changes. What do you thinl @pb866 ?

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  - only `nvar` values were tested, not covering the whole `JVS(:)` values.
+  - a strict equality (`== 0`) was used, while `abs < tiny` is safer.
+
+UPDATED: April 2019
+-------------------
+
+The changes listed above have been applied and documented with pull requests #1-#5 and #9
+in the __KPP-Mistra__ repository (https://github.com/MistraModel/KPP-Mistra).
+
+Additional changes by P. Brauer & R. Sommariva:
```
Comment (pb866) [inline CHANGELOG.md]:
```suggestion
Additional changes by P. Bräuer & R. Sommariva:
```

File: README.md [hunk 2]
```diff
@@ -1,94 +1,96 @@
...
+__KPP__ is a symbolic chemistry Kinetics PreProcessor, developed by
+V. Damian and A. Sandu with contribution by R. Sander
+(https://people.cs.vt.edu/~asandu/Software/Kpp).
+
+__KPP-Mistra__ is a custom version of KPP for the Mistra model
+(https://github.com/MistraModel/Mistra). It was developed starting
+from KPP v2.2.3 and includes all the bugfixes and modifications by
+R. Sander and J. Bock (https://bitbucket.org/gcst/kpp), plus some
+additional modifications specific to Mistra by J. Bock. See
+`CHANGELOG.md` for a summary of the changes.
```
Comment (pb866) [inline README.md]: You heavily use back ticks, which are really meant to quote code. I'd rather link the changelog file directly here, so people are forwarded to the corresponding file.
```suggestion
[changelog](CHANGELOG.md) for a summary of the changes.
```

File: README.md [hunk 3]
```diff
@@ -1,94 +1,96 @@
...
+
+The stable version of KPP-Mistra can be downloaded here:
+https://github.com/MistraModel/KPP-Mistra/releases.
 
 ******************************************************************************
 
-__To get started with KPP:__  Read user's manual (doc/kpp_UserManual.pdf)
+__To get started with KPP:__  Read user's manual (`doc/kpp_UserManual.pdf`)
```
Comment (pb866) [inline README.md]:
```suggestion
__To get started with KPP:__  Read [user's manual](doc/kpp_UserManual.pdf)
```

File: README.md [hunk 4]
```diff
@@ -1,94 +1,96 @@
...
+(https://people.cs.vt.edu/~asandu/Software/Kpp).
+
+__KPP-Mistra__ is a custom version of KPP for the Mistra model
+(https://github.com/MistraModel/Mistra). It was developed starting
+from KPP v2.2.3 and includes all the bugfixes and modifications by
+R. Sander and J. Bock (https://bitbucket.org/gcst/kpp), plus some
+additional modifications specific to Mistra by J. Bock. See
+`CHANGELOG.md` for a summary of the changes.
+
+The stable version of KPP-Mistra can be downloaded here:
```
Comment (pb866) [inline README.md]: Could be simplified to:
```suggestion
The stable version of KPP-Mistra can be downloaded [here](https://github.com/MistraModel/KPP-Mistra/releases).
```

But it's also okay to explicitly list the link.

File: README.md [hunk 5]
```diff
@@ -1,94 +1,96 @@
...
+https://github.com/MistraModel/KPP-Mistra/releases.
```
Comment (pb866) [inline README.md]:
```suggestion
```
Needs to be deleted, if changed above. (I just didn't know how to review 2 lines at once.)

**Comment by pb866** [comment]
I don't think, we need the diff file. Everyone can run a simple diff themselves or it is available from the git history and changelog(?).

**Comment by rs028** [comment]
When I did this, I followed the various commits on the bitbucket repo, but they are not necessarily in the same order as Josue and Rolf did them. So the history is preserved in a somewhat different order in the various PR.  The main issue is that it is not alwas clear why some changes were made but I can't do much about it (hence the memo #11).

Any other suggestion?

I will leave this open for a few days, in case something needs to be added after resolving https://github.com/MistraModel/Mistra/issues/10.

**Comment by pb866** [comment]
All good from my side. Feel free to merge. We can always add changes to the documentation later anyway.

File: README.md (hunk 2 above)
Comment (rs028) [reply README.md]: I normally use back ticks for variable names and pieces of code. Maybe the filenames should not be like that, but I think it helps the readability.

**Comment by rs028** [comment]
I think I will remove this:

```
If you have any problems please send the detailed error report and the machine environment to: sandu@cs.vt.edu
```

It may give the impression he is involved with this particular version. His email is in the header anyway so I think it is okay to delete the line.

**Comment by rs028** [comment]
I will merge this. When I am sure there are no other updates I will freeze everything in a new release. I don't think at this stage we need to work anymore on this.

**Pull request merged**